tanner/tests/test_dbutils.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+import asynctest
```

Comment:
do you use it? either add it to requirements or delete from here :)

Reply:
Yeah, I'll remove them since asynctest wasn't able to help much in mocking Postgres.